File: semgrep-core/tests/js/metavar_anno.js
// ERROR:
@Anno1
class Foo {
    foo() {
        var foo = 1;
    }
}

// ERROR:
@Anno1()
@Anno2()
class FooBar {
    foo() {
        var foo = 2;
    }
}

@AnnoVar("Bar")
class Bar {
    bar() {
        var bar = 1;
    }
}

class NoAnno {
    noanno() {
        var no_anno = 1;
    }
}

File: tests/patterns/java/metavar_anno.java
// ERROR:
@AnnoFoo
class Foo {
    int foo = 1;
}

// ERROR:
@Anno1
@Anno2
class FooBar {
    int foobar = 2;
}

@AnnoBar(bar = "bar")
class Bar {
    int bar = 1;
}

class NoAnno {
    int noanno = 2;
}
